### PR TITLE
zh: update docs about alert rules and handling

### PR DESCRIPTION
**Pull request by csuzhangxc**
### What is changed, added or deleted? (Required)

update docs about alert rules and handling

### Which TiDB Data Migration (DM) version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [x] v1.0 (TiDB DM 1.0 versions)

### CODE REVIEW

File: zh/monitor-a-dm-cluster.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ overview 下包含运行当前选定 task 的所有 DM-worker instance 的部分
 
 | metric 名称 | 说明 | 告警说明 | 告警级别 |
 |:----|:------------|:----|:----|
-| task state | 同步子任务的状态 | 当子任务状态处于 paused 超过 10 分钟时| critical |
+| task state | 同步子任务的状态 | 当子任务状态处于 paused 超过 20 分钟时| critical |
```
Comment (csuzhangxc) [inline zh/monitor-a-dm-cluster.md:37]: 我们 DM 告警规则里是写的 20 分钟。https://github.com/pingcap/dm/blob/83063141c5fd29058377336e90e448027be6206d/dm/dm-ansible/conf/dm_worker.rules.yml#L94

File: zh/handle-alerts.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+## Binlog replication 告警
+
+### `DM_sync_process_exists_with_error`
+
+当 Binlog replication 处理单元遇到错误时，会转为 paused 状态并立即触发该告警，此时需要参考[DM 故障诊断](error-handling.md#dm-故障诊断)进行处理。
+
+### `DM_binlog_file_gap_between_master_syncer`
+
+当 Binlog replication 处理单元已处理到的 binlog 文件个数落后于上游 MySQL/MariaDB 超过 1 个（不含 1 个）且持续 10 分钟时 DM 会触发该告警，些时需要参考[性能问题及处理方法](handle-performance-issues.md)对 Binlog replication 处理单元相关的性能问题进行排查与处理。
```
Comment (WangXiangUSTC) [inline zh/handle-alerts.md:65]:
```suggestion
当 Binlog replication 处理单元已处理到的 binlog 文件个数落后于上游 MySQL/MariaDB 超过 1 个（不含 1 个）且持续 10 分钟时 DM 会触发该告警，此时需要参考[性能问题及处理方法](handle-performance-issues.md)对 Binlog replication 处理单元相关的性能问题进行排查与处理。
```

File: zh/handle-alerts.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+### `DM_binlog_file_gap_between_relay_syncer`
+
+当 Binlog replication 处理单元已处理到的 binlog 文件个数落后于 relay log 处理单元超过 1 个（不含 1 个）且持续 10 分钟时 DM 会触发该告警，些时需要参考[性能问题及处理方法](handle-performance-issues.md)对 Binlog replication 处理单元相关的性能问题进行排查与处理。
```
Comment (WangXiangUSTC) [inline zh/handle-alerts.md:69]:
```suggestion
当 Binlog replication 处理单元已处理到的 binlog 文件个数落后于 relay log 处理单元超过 1 个（不含 1 个）且持续 10 分钟时 DM 会触发该告警，此时需要参考[性能问题及处理方法](handle-performance-issues.md)对 Binlog replication 处理单元相关的性能问题进行排查与处理。
```

File: zh/handle-alerts.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+当 relay log 处理单元在尝试从上游读取 binlog event 发生错误时，会转为 paused 状态并立即触发该告警，此时需要参考[DM 故障诊断](error-handling.md#dm-故障诊断)进行处理。
+
+### `DM_fail_to_write_relay_log`
+
+当 relay log 处理单元在尝试将 binlog event 写入 relay log 文件发生错误时，会转为 paused 状态并立即触发该告警，此时需要参考[DM 故障诊断](error-handling.md#dm-故障诊断)进行处理。
+
+### `DM_binlog_file_gap_between_master_relay`
+
+当 relay log 处理单元已拉取到的 binlog 文件个数落后于上游 MySQL/MariaDB 超过 1 个（不含 1 个）且持续 10 分钟时会触发该告警，些时需要参考[性能问题及处理方法](handle-performance-issues.md)对 relay log 处理单元相关的性能问题进行排查与处理。
```
Comment (WangXiangUSTC) [inline zh/handle-alerts.md:45]: 是指拉取最新 binlog 文件的序号比上游最新 binlog 文件的序号少1个吧

Comment (csuzhangxc) [reply zh/handle-alerts.md:45]: 假如上游是 `mysql-bin.000003`, relay 当前最新是 `mysql-bin.000001`，就告警

Comment (WangXiangUSTC) [reply zh/handle-alerts.md:45]: 所以感觉 `已拉取到的 binlog 文件个数落后于上游 MySQL/MariaDB 超过 1 个（不含 1 个）` 不太精确

Comment (csuzhangxc) [reply zh/handle-alerts.md]: 稍微改了一下，再看看。或者有没有更好的表述。

**Comment by WangXiangUSTC** [comment]
LGTM

File: zh/handle-alerts.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+
+# 告警处理
+
+本文档介绍 DM 中各主要告警信息的处理方法。
+
+## 任务状态告警
+
+### `DM_task_state`
+
+当 DM-worker 内有子任务的状态处于 paused 超过 20 分钟时会触发该告警，此时需要参考[DM 故障诊断](error-handling.md#dm-故障诊断)进行处理。
```
Comment (toutdesuite) [inline zh/handle-alerts.md:15]:
```suggestion
当 DM-worker 内有子任务处于 `Paused` 状态超过 20 分钟时会触发该告警，此时需要参考[DM 故障诊断](error-handling.md#dm-故障诊断)进行处理。
```

File: zh/handle-alerts.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+## relay log 告警
+
+### `DM_relay_process_exits_with_error`
+
+当 relay log 处理单元遇到错误时，会转为 paused 状态并立即触发该告警，此时需要参考[DM 故障诊断](error-handling.md#dm-故障诊断)进行处理。
```
Comment (toutdesuite) [inline zh/handle-alerts.md:21]:
```suggestion
当 relay log 处理单元遇到错误时，会转为 `Paused` 状态并立即触发该告警，此时需要参考[DM 故障诊断](error-handling.md#dm-故障诊断)进行处理。
```

File: zh/handle-alerts.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+- 尝试调整 relay log 的[自动清理策略](relay-log.md#自动数据清理)或执行[手动清理](relay-log.md#手动数据清理)。
+- 尝试[迁移 DM-worker 实例](cluster-operations.md#替换迁移-dm-worker-实例)到其他可用容量充足的磁盘上。
+
+### `DM_relay_log_data_corruption`
+
+当 relay log 处理单元在校验从上游读取到的 binlog event 且发现 checksum 信息异常时会转为 paused 状态并立即触发告警，此时需要参考[DM 故障诊断](error-handling.md#dm-故障诊断)进行处理。
+
+### `DM_fail_to_read_binlog_from_master`
+
+当 relay log 处理单元在尝试从上游读取 binlog event 发生错误时，会转为 paused 状态并立即触发该告警，此时需要参考[DM 故障诊断](error-handling.md#dm-故障诊断)进行处理。
```
Comment (toutdesuite) [inline zh/handle-alerts.md:37]:
```suggestion
当 relay log 处理单元在尝试从上游读取 binlog event 发生错误时，会转为 `Paused` 状态并立即触发该告警，此时需要参考[DM 故障诊断](error-handling.md#dm-故障诊断)进行处理。
```

File: zh/handle-alerts.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+
+### `DM_fail_to_write_relay_log`
+
+当 relay log 处理单元在尝试将 binlog event 写入 relay log 文件发生错误时，会转为 paused 状态并立即触发该告警，此时需要参考[DM 故障诊断](error-handling.md#dm-故障诊断)进行处理。
```
Comment (toutdesuite) [inline zh/handle-alerts.md:41]:
```suggestion
当 relay log 处理单元在尝试将 binlog event 写入 relay log 文件发生错误时，会转为 `Paused` 状态并立即触发该告警，此时需要参考 [DM 故障诊断](error-handling.md#dm-故障诊断)进行处理。
```

File: zh/handle-alerts.md [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+
+### `DM_binlog_file_gap_between_master_relay`
+
+当 relay log 处理单元已拉取到的最新的 binlog 文件个数落后于当前上游 MySQL/MariaDB 超过 1 个（不含 1 个）且持续 10 分钟时会触发该告警，此时需要参考[性能问题及处理方法](handle-performance-issues.md)对 relay log 处理单元相关的性能问题进行排查与处理。
+
+## Dump/Load 告警
+
+### `DM_dump_process_exists_with_error`
+
+当 dump 处理单元遇到错误时，会转为 paused 状态并立即触发该告警，此时需要参考[DM 故障诊断](error-handling.md#dm-故障诊断)进行处理。
```
Comment (toutdesuite) [inline zh/handle-alerts.md:51]:
```suggestion
当 Dump 处理单元遇到错误时，会转为 `Paused` 状态并立即触发该告警，此时需要参考[DM 故障诊断](error-handling.md#dm-故障诊断)进行处理。
```

File: zh/handle-alerts.md [hunk 10]
```diff
@@ -0,0 +1,69 @@
...
+
+### `DM_load_process_exists_with_error`
+
+当 load 处理单元遇到错误时，会转为 paused 状态并立即触发该告警，此时需要参考[DM 故障诊断](error-handling.md#dm-故障诊断)进行处理。
```
Comment (toutdesuite) [inline zh/handle-alerts.md:55]:
```suggestion
当 Load 处理单元遇到错误时，会转为 `Paused` 状态并立即触发该告警，此时需要参考 [DM 故障诊断](error-handling.md#dm-故障诊断)进行处理。
```

File: zh/handle-alerts.md [hunk 11]
```diff
@@ -0,0 +1,69 @@
...
+
+## Binlog replication 告警
+
+### `DM_sync_process_exists_with_error`
+
+当 Binlog replication 处理单元遇到错误时，会转为 paused 状态并立即触发该告警，此时需要参考[DM 故障诊断](error-handling.md#dm-故障诊断)进行处理。
```
Comment (toutdesuite) [inline zh/handle-alerts.md:61]: ditto 英文与中文之间空一格

Comment (csuzhangxc) [reply zh/handle-alerts.md]: 在 https://github.com/pingcap/docs-dm/pull/93/commits/a4cfc1655b3422fed690d613a7a42f5b651bbd95 加了

**Comment by miaoqingli** [comment]
LGTM

File: zh/handle-alerts.md [hunk 12]
```diff
@@ -0,0 +1,69 @@
...
+
+当 relay log 所在磁盘的剩余可用容量小于 10G 时会触发该告警，对应的处理方法包括：
+
+- 手动清理该磁盘上其他无用数据以增加可用容量。
+- 尝试调整 relay log 的[自动清理策略](relay-log.md#自动数据清理)或执行[手动清理](relay-log.md#手动数据清理)。
+- 尝试[迁移 DM-worker 实例](cluster-operations.md#替换迁移-dm-worker-实例)到其他可用容量充足的磁盘上。
+
+### `DM_relay_log_data_corruption`
+
+当 relay log 处理单元在校验从上游读取到的 binlog event 且发现 checksum 信息异常时会转为 paused 状态并立即触发告警，此时需要参考 [DM 故障诊断](error-handling.md#dm-故障诊断)进行处理。
```
Comment (toutdesuite) [inline zh/handle-alerts.md:33]:
```suggestion
当 relay log 处理单元在校验从上游读取到的 binlog event 且发现 checksum 信息异常时会转为 `Paused` 状态并立即触发告警，此时需要参考 [DM 故障诊断](error-handling.md#dm-故障诊断)进行处理。
```

File: zh/handle-alerts.md [hunk 13]
```diff
@@ -0,0 +1,69 @@
...
+
+### `DM_load_process_exists_with_error`
+
+当 Load 处理单元遇到错误时，会转为 `Paused` 状态并立即触发该告警，此时需要参考 [DM 故障诊断](error-handling.md#dm-故障诊断)进行处理。
+
+## Binlog replication 告警
+
+### `DM_sync_process_exists_with_error`
+
+当 Binlog replication 处理单元遇到错误时，会转为 paused 状态并立即触发该告警，此时需要参考 [DM 故障诊断](error-handling.md#dm-故障诊断)进行处理。
```
Comment (toutdesuite) [inline zh/handle-alerts.md:61]:
```suggestion
当 Binlog replication 处理单元遇到错误时，会转为 `Paused` 状态并立即触发该告警，此时需要参考 [DM 故障诊断](error-handling.md#dm-故障诊断)进行处理。
```

File: zh/monitor-a-dm-cluster.md (hunk 1 above)
Comment (toutdesuite) [inline zh/monitor-a-dm-cluster.md:37]:
```suggestion
| task state | 同步子任务的状态 | 当子任务状态处于 `Paused` 超过 20 分钟时| critical |
```

**Pull request merged**